packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -30,7 +37,31 @@ const attributeRouter = (
         M2M_ROLE,
         SUPPORT_ROLE,
       ]),
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        try {
+          const { limit, offset, kinds, name, origin } = req.query;
+
+          const attributes = await readModelService.getAttributes(
+            {
+              kinds: kinds.map(apiAttributeKindToAttributeKind),
+              name,
+              origin,
+            },
+            offset,
+            limit
+          );
+
+          return res
+            .status(200)
+            .json({
+              results: attributes.results.map(attributeToApiAttribute),
+              totalCount: attributes.totalCount,
+            })
+            .end();
+        } catch (error) {
+          return res.status(500).end();
```

Comment:
Why not use this:

```suggestion
          const errorRes: ApiError = makeApiError(error);
          return res.status(errorRes.status).json(errorRes).end();
```

Reply:
> Why not use this:

In order to use it here we have to include the following response in the open API:

```
        '500':
          description: Internal server error
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Problem'
```
        
Otherwise the types differ and it will not compile

Reply:
Let's approach this problem in another PR.